pyproject.toml

```diff
@@ -33,7 +33,7 @@ dynamic = ["version"]
 dependencies = [
     "dvc>=2.58.0",
     "dvc-render>=0.5.0,<1.0",
-    "dvc-studio-client>=0.10.0,<1",
+    "dvc-studio-client>=0.13.0,<1",
```

Comment:
Guess we would need a new dvc-studio-client release and to bump the version here